src/pages/privacy.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import AppMinimal from '../components/layout/AppMinimal';
+import Seo from '../components/molecules/Seo';
+
+import PrivacyBox from '../components/organims/PrivacyBox';
+
+class Privacy extends React.Component {
+  constructor(props) {
+    super(props);
+
+    // Init State
+    this.state = {
+      isClient: false,
+    };
+
+  }
+
+  componentDidMount() {
+    this.setState({
+      isClient: true,
+    });
+  }
+
+  render() {
+    const { isClient } = this.state;
```

Comment:
Could have been worth adding an explanatory comment of the technique used here to mix SSR and CSR